mobile/src/main/java/rs/readahead/washington/mobile/views/settings/GeneralSettings.kt

```diff
@@ -31,4 +46,14 @@ class GeneralSettings : Fragment() {
 
         return view
     }
+
+    private fun setLanguageSetting() {
+        val language = LocaleManager.getInstance().languageSetting
+        if (language != null) {
+            val locale = Locale(language)
```

Comment:
I would suggest to get rid of null check when we use kotlin, maybe we can use some scope functions : `  LocaleManager.getInstance().languageSetting?.let { language ->
            val locale = Locale(language)
            languageSetting?.text = StringUtils.capitalize(locale.displayName, locale) 
        }?.run {
            languageSetting?.setText(R.string.settings_lang_select_default)
        }`

Reply:
Although I still don't get along well with all the possibilities of Kotlin, I generally agree. But in this case, null is also information that the language has not been set, ie we use device default.

Reply:
Indeed, that's why we executed `?.run {}` to handle the case when ever we have null value.

Reply:
Of course, only the current code seems much cleaner to me. But it's ok for me anyway.